src/components/Actions/Actions.vue

```diff
@@ -360,6 +360,52 @@ export default {
 </script>
 ```
 
+### Scoped slot
+
+```vue
+<template>
+    <div>
+        <input ref="input" />
+        <Actions>
+            <template v-slot:default="{ clearFocusTrap }">
+				<ActionButton @click="focusInput(clearFocusTrap)" :close-after-click="true">
+					<template #icon>
+						<Delete :size="20" />
```

Comment:
If we add this example to the docs, I would chose a better icon :slightly_smiling_face:

Reply:
Maybe `Plus`?